cmd/livepeer/starter/kafka.go

```diff
@@ -0,0 +1,25 @@
+package starter
+
+import (
+	"github.com/golang/glog"
+	lpmon "github.com/livepeer/go-livepeer/monitor"
+)
+
+func startKafkaProducer(cfg LivepeerConfig) error {
+	if *cfg.KafkaBootstrapServers == "" || *cfg.KafkaUsername == "" || *cfg.KafkaPassword == "" || *cfg.KafkaGatewayTopic == "" {
+		glog.Warning("not starting Kafka producer as producer config values aren't present")
```

Comment:
do you need to return here?

Reply:
No, starting without pushing to Kafka is fine if it's not configured